CMakeLists.txt

```diff
@@ -637,6 +637,7 @@ elseif(ANDROID)
   )
 
   if(LOVR_BUILD_EXE)
+    find_package(Java)
```

Comment:
This already exists on line 59, which confuses me

Reply:
Then I think this one is a merge error specific to my branch.

I will retest and remove this line.